### PR TITLE
Feat/shop

### PR DESCRIPTION
**Pull request by eltfshr**
fix parameter order of error handling function in server.js

### CODE REVIEW

**Comment by riflowth** [comment]
@eltfshr https://expressjs.com/en/guide/error-handling.html

**Comment by riflowth** [comment]
Thanks for PR 😃

File: server/models/shop.js [hunk 1]
```diff
@@ -9,4 +8,19 @@ exports.getShops = async () => {
...
     let shop = await knex.select().from('shops').where({id:id});
     return JSON.parse(JSON.stringify(shop[0]));
+}
+
+exports.getReview = async (shopId) => {
+    let reviews = await knex.select('rating','review').from('reviews').where({shop_id:shopId});
+    return JSON.parse(JSON.stringify(reviews));
+}
+
+exports.writeReview = async (rating,userId,review,shopId) => {
```
Comment (riflowth) [inline server/models/shop.js]: How different from api.js ?

Comment (eltfshr) [reply server/models/shop.js]: api.js only handles the posted request body then uses the function from line 18 of shop.js to write to database

**Comment by riflowth** [comment]
Thanks for PR na **Pun** 👍

**Pull request merged**